content/blog/hackathon-2022-wazo-matrix-bridge.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: Hackathon 2022 - Bridging Wazo and Matrix
+Date: 2022-10-20 12:00:00
+Author: Jesse Sopel
+Category: Hackathon
+Tags: Chat, Python, Matrix, Bridge, Hackathon
+Slug: hackathon-2022-bridging-wazo-and-matrix
+Status: published
+---
+
+Every year Wazo organizes a Hackathon with their employees, and as many of us as possible get together to work on some fun projects and see what we can learn.
+There were multiple teams working on different projects. Our team consisted of Charles, Francis, Francois and I (Jesse).
```

Comment:
maybe you could put the links to their github profiles :)

Reply:
Good idea. I'll try and find them.

Reply:
Added!